geode-core/src/main/java/org/apache/geode/management/internal/beans/MemberMBeanBridge.java

```diff
@@ -430,14 +428,8 @@ public MemberMBeanBridge init() {
     if (PureJavaMode.osStatsAreAvailable()) {
       Statistics[] systemStats = null;
 
-      if (HostStatHelper.isSolaris()) {
-        systemStats = system.findStatisticsByType(SolarisSystemStats.getType());
-      } else if (HostStatHelper.isLinux()) {
+      if (HostStatHelper.isLinux()) {
```

Comment:
This change should not have been needed. PureJavaMode.osStatsAreAvailable should now only return true if isLinux. Since os stat availability no longer has anything to do with PureJavaMode, I think you should move osStatsAreAvailable to the HostStatHelper. HostStatHelper is the class that should manage everything having to do with os stats. Also, have it be something like HostStatHelper.osStatsSupported instead of isLinux. In the future we might want to support os stats on platforms in addition to Linux so code outside of HostStatHelper should not be testing isLinux to decide if os stats are available.